### PR TITLE
New viewer

### PR DESCRIPTION
**Pull request by kerrj**
Integrate changes from the new viewer in Nerfstudio and update documentation to reflect them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,21 @@
...
-
-### 4. Run `ns-install-cli`
+### 0. Install Nerfstudio dependencies
+[Follow these instructions](https://docs.nerf.studio/en/latest/quickstart/installation.html) up to and including "tinycudann" to install dependencies and create an environment
+### 1. Clone this repo
+`git clone https://github.com/kerrj/lerf`
+### 2. Install this repo as a python package
+Navigate to this folder and run `python -m pip install -e .`
+
+### 3. Run `ns-install-cli`
```
Comment (chungmin99) [inline README.md:18]: This isn't true (about `ns-install-cli`); only the LERF package installation (`pip install`) is required for `ns-train` to register the LERF command (per https://docs.nerf.studio/en/latest/developer_guides/new_methods.html) 

Other than that, I agree that running `ns-install-cli` is best practice. 

Also, did you ever figure out why the `pip install git+https://..."` doesn't work? (not that it's critical)

File: lerf/encoders/openclip_encoder.py [hunk 2]
```diff
@@ -67,8 +73,8 @@ def name(self) -> str:
     def embedding_dim(self) -> int:
         return self.config.clip_n_dims
 
-    def set_positives(self, text_list):
-        self.positives = text_list
+    def set_positives(self, handle):
```
Comment (chungmin99) [inline lerf/encoders/openclip_encoder.py:76]: Two questions (changing `set_positives`'s args to `text_list` -> `handle`):
1.  Is `handle` ever used in `set_positives`? Is this variable necessary?
2. Does this remove the ability to feed in a text prompt directly (via code w/o viewer)? Or is there a way to do this via viser? My main worry is that this code needs to be refactored for `ns-render` later on.

**Review by chungmin99 — COMMENTED** [review]
Seems mostly good, but had a couple of comments

File: lerf/encoders/openclip_encoder.py (hunk 2 above)
Comment (kerrj) [reply lerf/encoders/openclip_encoder.py:76]: true yea should change it to be more modular

**Comment by chungmin99** [comment]
👍

**Review by chungmin99 — APPROVED** [review]
💯

**Pull request merged**